### PR TITLE
UI : New Data profiler Layout.

### PR DESCRIPTION
**Pull request by Sachin-chaurasiya**
### Describe your changes :

I worked on the new data profiler layout. for more details see #1311 

#
### Type of change :

- [x] Improvement
- [x] New feature

#
### Frontend Preview (Screenshots) :
image

image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [x] I have performed a self-review of my own. 
- [x] I have tagged my reviewers below.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@shahsank3t, @darth-coder00

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/TableProfiler/TableProfiler.component.tsx [hunk 1]
```diff
@@ -120,84 +147,128 @@ const TableProfiler = ({ tableProfiles, columns }: Props) => {
...
+                        )}
+                        <div className="tw-self-center tw-ml-5">
+                          <Button
+                            className="tw-px-2 tw-py-0.5 tw-rounded tw-border-grey-muted"
+                            size="custom"
+                            type="button"
+                            variant="outlined"
+                            onClick={() => quilityTestFormHandler(6, 'column')}>
+                            Add Test
+                          </Button>
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/components/TableProfiler/TableProfiler.component.tsx:266]: We must restrict the non-owner and non-admin person for adding test case. and also i think it should have name `qualityTestFormHandler`

**Review by ShaileshParmar11 — CHANGES_REQUESTED** [review]
left one comment. other that that LGTM

**Review by ShaileshParmar11 — APPROVED** [review]
LGMT!!

**Review by darth-coder00 — APPROVED** [review]
LGTM!

**Pull request merged**